Dalamud/Utility/Signatures/SignatureHelper.cs

```diff
@@ -24,7 +26,8 @@ public static class SignatureHelper
     /// </summary>
     /// <param name="self">The object to initialise.</param>
     /// <param name="log">If warnings should be logged using <see cref="PluginLog"/>.</param>
-    public static void Initialise(object self, bool log = true)
+    /// <returns>Collection of created IDalamudHooks.</returns>
+    internal static IEnumerable<IDalamudHook> Initialise(object self, bool log = true)
```

Comment:
This method is currently also used to initialize actual signatures as well, e.g.:

```cs
public class GameFunctions {
    private delegate byte IsQuestCompletedDelegate(ushort questId);

    [Signature("E8 ?? ?? ?? ?? 41 88 84 2C")]
    private readonly IsQuestCompletedDelegate? _isQuestCompleted = null;

    public GameFunctions() {
        SignatureHelper.Initialise(this);
    }
}
```

Given these aren't hooks (and, as such, don't need to be tracked), how are we going to be handling these cases? A precursory read suggests that they'll be created anyways (and just won't be returned), but documentation does not particularly support this point - see other review comments.